todo-app/src/App.js

```diff
@@ -1,13 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
+import {AppTask} from './AppTask';
+import {AppDate} from './AppDate';
 
-class App extends Component {
-  render() {
-    return (
-      <div>
-        Hello World!
-      </div>
-    );
-  }
+export class Todo extends React.Component{
+    Constructor(){
+        
+    }
+    
+    render(){
+        return (
+            <div className = "todoApp">
+            <h1>Todo App</h1>
+            <AppDate tasks={this.props.tasks}/>
+            <AppTask tasks={this.props.tasks}/>
```

Comment:
What was your reasoning for splitting the tasks and dates into two separate children like this? For now they render on top of each other and are "together", but what if we had a very long list of 100 items? It's fine to split date and task content into two separate components - say we call them `TodoTask` and `TodoDate` - but they should then be wrapped in a higher-order component just called `Todo`, and you can loop through the `Todo` components instead.